service/src/main/java/tds/session/web/advice/ExceptionAdvice.java

```diff
@@ -0,0 +1,46 @@
+package tds.session.web.advice;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+import tds.common.web.exceptions.NotFoundException;
+import tds.common.web.resources.ExceptionMessageResource;
+
+@ControllerAdvice
+class ExceptionAdvice {
```

Comment:
Shouldn't this be part of common so we don't have to define it in each microservice and we can have a common approach to handling exceptions?